API/API.ml

```diff
@@ -185,13 +185,15 @@ module Entries =
     type inline = int option
     type 'a proof_output = Constr.constr Univ.in_universe_context_set * 'a
     type 'a const_entry_body = 'a proof_output Future.computation
+    type constant_universes_entry = Entries.constant_universes_entry =
+      | Monomorphic_const_entry of Univ.UContext.t
+      | Polymorphic_const_entry of Univ.UContext.t
     type 'a definition_entry = 'a Entries.definition_entry =
                                  { const_entry_body   : 'a const_entry_body;
                                    const_entry_secctx : Context.Named.t option;
                                    const_entry_feedback : Stateid.t option;
                                    const_entry_type        : Term.types option;
-                                   const_entry_polymorphic : bool;
```

Comment:
Indeed, the goal is even to get nothing in the monomorphic case, and an abstract context in the polymorphic case.